pinot-spi/src/test/java/org/apache/pinot/spi/ingestion/batch/IngestionJobLauncherTest.java

```diff
@@ -48,6 +48,7 @@ public void testIngestionJobLauncherWithTemplate() {
         null, context, _defaultEnvironmentValues);
     Assert.assertEquals(spec.getInputDirURI(), "file:///path/to/input/2020/05/06");
     Assert.assertEquals(spec.getOutputDirURI(), "file:///path/to/output/2020/05/06");
+    Assert.assertFalse(spec.isSearchRecursively());
```

Comment:
make a comment explicitly saying this is set to false in yaml config explicitly

Reply:
Updated the comment